### ISSUE TITLE
elevated serum insulin–to–C-peptide ratio 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7674]
**Preferred term label:**
Elevated circulating insulin–to–C-peptide ratio 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Increased ratio of insulin to C-peptide in the blood circulation

Insulin and C-peptide are secreted into portal vein in equimolar amounts, but the serum ratio is normally from 1:5 to 1:15 due to removal of approximately 50% of insulin from blood during initial passage through the liver.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

609968 
PMID: 15161766   10/10
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
done
